src/public/app/widgets/mobile_widgets/mobile_global_buttons.js

```diff
@@ -21,6 +26,14 @@ const WIDGET_TPL = `
 
     <a data-trigger-command="scrollToActiveNote" title="Scroll to active note" class="icon-action bx bx-crosshair"></a>
 
+    <div class="dropdown">
+        <a title="Plugin buttons" class="icon-action bx bx-chip dropdown-toggle" data-toggle="dropdown"></a>
```

Comment:
For consistency, I would prefer `bx-extension` should be used since that's also used for conceptually similar "render note".

Reply:
Updated